diff --git a/conformance/interfaces/pthread_cond_broadcast/4-2.c b/conformance/interfaces/pthread_cond_broadcast/4-2.c
--- a/conformance/interfaces/pthread_cond_broadcast/4-2.c
+++ b/conformance/interfaces/pthread_cond_broadcast/4-2.c
@@ -273,4 +273,9 @@ int main (int argc, char * argv[])
 	do { do_it=0; }
 	while (do_it);
 	
+	#ifdef WITH_SYNCHRO
+	(void)sem_post( &semsig1 );
+	(void)sem_post( &semsig2 );
+	#endif
+
 	if ((ret = pthread_join(th_sig1, NULL)))
